book/content/binderhub/binderhub.md

```diff
@@ -0,0 +1,159 @@
+- [How does a BinderHub work?](#how-does-a-binderhub-work)
+  - [Compute Resources](#compute-resources)
+  - [Kubernetes](#kubernetes)
+  - [Helm](#helm)
+  - [JupyterHub](#jupyterhub)
+  - [repo2docker](#repo2docker)
+  - [What happens when a Binder link is clicked?](#what-happens-when-a-binder-link-is-clicked)
+- [Why would you build your own BinderHub?](#why-would-you-build-your-own-binderhub)
+  - [Issues you may face when deploying a BinderHub](#issues-you-may-face-when-deploying-a-binderhub)
+- [Further reading](#further-reading)
+- [Definitions/glossary](#definitionsglossary)
+- [Bibliography](#bibliography)
+
+## Summary
+
+This chapter will discuss [BinderHub](https://binderhub.readthedocs.io/en/latest/index.html), which is the cloud technology powering [Binder](https://mybinder.readthedocs.io/en/latest/).
+We will cover the technologies and tools that BinderHub utilises and the resources you will need to setup your own BinderHub.
+
+This chapter is primarily aimed at Research Software Engineers and IT Services who wish to provide a BinderHub as a service to a group of researchers.
+Though anyone can build a BinderHub.
+
+## How this will help you/ why this is useful
+
+Reading this chapter will give you a clearer picture of how Binder services (such as [mybinder.org](https://mybinder.org)) operate, the technologies powering BinderHub and how they interact with one another.
+This chapter also covers reasons why you might build your own BinderHub, rather than using the public service.
+
+## What is BinderHub and why is it good for Reproducibility?
+
+[BinderHub](https://binderhub.readthedocs.io/en/latest/index.html) is a cloud-based technology that can launch a repository of code (from GitHub, GitLab,...) in a browser window such that the code can be executed and interacted with.
+A unique URL is generated allowing the interactive code to be easily shared.
+
+The purpose of these Binder instances is to promote reproducibility in research projects by encouraging researchers to document their software dependencies and produce fun, interactive environments!
+
+Binder, as a user interface, is useful for reproducibility because the code needs to be version controlled and the computational environment needs to be documented in order to benefit from the functionality of Binder.
+Each change to the code repository also forces a new build of the Binder instance.
+This acts as a proxy for continuous integration of the computational environment as the Binder instance will break if the configuration file is not updated.
+
+Learn more about Continuous Integration in [this chapter](/continuous_integration/continuous_integration).
+
+## How does a BinderHub work?
+
+BinderHub relies on different tools and resources in order to create and launch the Binder instances.
+
+A high-level explanation of the BinderHub architecture can be found [here](https://binderhub.readthedocs.io/en/latest/overview.html).
```

Comment:
```suggestion
For more information, see this [high-level explanation of the BinderHub architecture](https://binderhub.readthedocs.io/en/latest/overview.html).
```

I've found it's better to link content-phrases (like above) instead of words like "here" or "this link" so that people notice them better